manifests/p/Phoner/PhonerLite/2.93.0/Phoner.PhonerLite.yaml

```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: Phoner.PhonerLite
+Publisher: Heiko Sommerfeldt
+PackageName: PhonerLite
+Moniker: PhonerLite
+PackageVersion: 2.93.0
```

Comment:
```suggestion
PackageVersion: 2.93
```

Reply:
**You'll have to update the version folder to match `PackageVersion`**

This change was made to match the application's version with the one shown in Control Panel.